observation_portal/observations/viewsets.py

```diff
@@ -79,49 +84,56 @@ def filters(self, request):
                 })
             else:
                 obs_filter_options['fields'].append(filter_name)
-        return Response(obs_filter_options, status=200)
+
+        response_serializer = self.get_response_serializer(data=obs_filter_options)
+        if response_serializer.is_valid():
+            return Response(response_serializer.validated_data, status=200)
+        else:
+            return Response(response_serializer.errors, status=400)
 
     @action(detail=False, methods=['post'])
     def cancel(self, request):
         """
```

Comment:
just an fyi I updated this method a bit recently in main when you pull it in, hopefully it will merge smoothly